cmd/init.go

```diff
@@ -288,7 +288,6 @@ func Init() *cli.Command {
 
 			successPrinter.Printf("\n\nA new '%s' pipeline created successfully in folder '%s'.\n", templateName, inputPath)
 			infoPrinter.Println("\nYou can run the following commands to get started:")
-			infoPrinter.Printf("\n    cd %s\n", inputPath)
```

Comment:
I think it makes sense to delete this. I think it might be good to add here
infoPrinter.Printf(" bruin run path\n\n") path is either just the pipeline name or bruin/pipeline-name